utils/url.h

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (C) 2019-2020 BiiLabs Co., Ltd. and Contributors
+ * All Rights Reserved.
+ * This is free software; you can redistribute it and/or modify it under the
+ * terms of the MIT license. A copy of the license can be found in the file
+ * "LICENSE" at the root of this distribution.
+ */
+
+#ifndef UTILS_URL_H
+#define UTILS_URL_H
+
+#include "defined_error.h"
+
+/**
+ * @brief send message via http protocol
+ *
+ * @param[in] host http host
+ * @param[in] port http port
+ * @param[in] api http api
```

Comment:
The descriptions about `api` is misleading. You have to explain with informative wording.